.travis.d/templates/deployment.yaml.erb

```diff
@@ -6,6 +6,11 @@ metadata:
     name: <%= app['uid'] %>
     heritage: Biodomes
 spec:
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
```

Comment:
Shouldn't this be `1 + ((app['deployment'] and app['deployment']['replicas']) or 1)` ?